src/freenet/clients/http/QueueToadlet.java

```diff
@@ -1978,15 +2019,32 @@ private void selectLocation(HTMLNode node) {
 	 *            The last activity of the request
 	 * @return The created table cell HTML node
 	 */
-	private HTMLNode createLastActivityCell(long now, long lastActivity) {
+	private HTMLNode createLastActivityCell(long now, Date lastActivity) {
 		HTMLNode lastActivityCell = new HTMLNode("td", "class", "request-last-activity");
-		if (lastActivity == 0) {
+		if (lastActivity == null) {
+            // This is "unknown"  instead of "never" because the backend of RequestStatus always
+            // tries to initialize lastActivity to current time, it should probably never be null.
```

Comment:
That sounds like it makes that an error, not just "unknown." Is that worthy of logging?